protocol/requests/create_account.go

```diff
@@ -15,6 +15,7 @@ type CreateAccount struct {
 	// BackupDisabledDataDir is the directory where backup is disabled
 	BackupDisabledDataDir string `json:"backupDisabledDataDir"`
 
+	DeviceName         string `json"deviceName"`
```

Comment:
Typo, Should be `json:"deviceName"`

Reply:
You are right, though what is weird is that it works anyway.

Reply:
https://cs.opensource.google/go/go/+/refs/tags/go1.20.5:src/reflect/type.go;l=1221 ok, the colon is not really needed by design, no miracles, the value will be detected by opening double quote https://cs.opensource.google/go/go/+/refs/tags/go1.20.5:src/reflect/type.go;l=1221

Reply:
I think it's better to be consistent.

Reply:
well yep it is already fixed some time ago, I just wanted to verify whether what I tested could actually work